client/executor/src/wasm_runtime.rs

```diff
@@ -329,6 +329,7 @@ pub fn create_wasm_runtime_with_code(
 					semantics: sc_executor_wasmtime::Semantics {
 						fast_instance_reuse: true,
 						stack_depth_metering: false,
+						canonicalize_nans: false,
```

Comment:
There is no code (apart from tests) which sets this to true. Is that correct?

Reply:
Yes, this is only for the PVF execution. Check out the companion PR!